dist/openshift/cincinnati.yaml

```diff
@@ -144,8 +144,16 @@ objects:
                 - name: status-pe
                   containerPort: ${{PE_STATUS_PORT}}
               livenessProbe:
-                tcpSocket:
-                  port: ${{PE_PORT}}
+                httpGet:
+                  path: /livez
+                  port: ${{GB_STATUS_PORT}}
+                initialDelaySeconds: 60
+                periodSeconds: 30
+                timeoutSeconds: 3
```

Comment:
3 as timeout seconds might be less. I wonder if we can increase this to 5 to 10 seconds to reduce the churn. but I am fine with starting with 3 and then increase if required.